2-mandatory/5-reading-list.js

```diff
@@ -47,7 +73,12 @@ As an example for this exercise, you might do the following steps
 - Modify the books so that they have the correct alreadyRead value
 - All tests should turn green!!
 **/
-
+// function logBooks() {
+//   for (let book of books){
+//     if (book.alreadyRead == true){console.log(`You've already read ${book.title} by ${book.author}`)}
```

Comment:
no need to declare `== true` in this case?

Reply:
good, I see. removed it